app/src/main/res/values-de/strings.xml

```diff
@@ -44,29 +44,29 @@
     <string name="notification_settings_title">Benachrichtigung über eingehende Dateien anzeigen</string>
     <string name="overwrite_settings_title">Überschreiben erlauben</string>
     <string name="accept_settings_title">Transfer automatisch akzeptieren</string>
-    <string name="accept_setting_summary_on">Der Transfer wird automatisch akzeptiert</string>
-    <string name="accept_setting_summary_off">Jeder Transfer muss von Dir akzeptiert werden</string>
-    <string name="background_settings_title">Im Hintergrund ausführend</string>
+    <string name="accept_setting_summary_on">Der Transfer wird automatisch angenommen</string>
+    <string name="accept_setting_summary_off">Jeder Transfer muss von Dir bestätigt werden</string>
+    <string name="background_settings_title">Im Hintergrund ausführen</string>
     <string name="boot_settings_title">Automatisch starten</string>
     <string name="export_log_settings_tile">Debug-Protokoll in Datei schreiben</string>
     <string name="group_code_settings_title">Gruppenschlüssel</string>
     <string name="port_settings_title">Port</string>
     <string name="theme_settings_title">Thema</string>
-    <string name="notification_settings_summary">Dies wird möglicherweise durch Android\'s Benachrichtigungseinstellungen beeinflusst</string>
+    <string name="notification_settings_summary">Dies wird möglicherweise durch Androids Benachrichtigungseinstellungen beeinflusst</string>
     <string name="port_range_warning">Port muss zwischen 1024 und 65535 liegen</string>
     <string name="requires_restart_warning">Die Änderung dieser Einstellung erfordert einen Neustart der Anwendung</string>
-    <string name="overwrite_settings_summary_on">Eingehende Dateien mit einem Namen der bereits existiert, überschreiben die existierenden</string>
-    <string name="overwrite_settings_summary_off">Eingehende Dateien mit einem Namen der bereits existiert, werden umbenannt</string>
+    <string name="overwrite_settings_summary_on">Eingehende Dateien mit einem Namen, der bereits existiert, überschreiben die bestehenden</string>
+    <string name="overwrite_settings_summary_off">Eingehende Dateien mit einem Namen, der bereits existiert, werden umbenannt</string>
```

Comment:
@uDEV2019 wrote, --> sphenotux comments
Line 58 and 59:
You set the comma wrong. --> Disagree
With your added comma, the sentence makes no sense anymore, because the words between the comma are now as if they are not there. German comma rules are complicated, --> ;-) yes
but necessary for german. --> not necessarily right. The section "der bereits existiert" is an **addition** to the previous described "Eingehende Dateien"

What about
"Eingehende Dateien mit einem existierenden Namen werden umbenannt" - or would that be too far away from English source? Even shorter:
"Eingehende Dateien überschreiben bestehende"
"Eingehende Dateien werden umbenannt"

Reply:
In addition: Eingehende Dateien mit einem Namen, WELCHER bereits existiert, überschreiben die vorhandenden would be correct as well. So we put a comma after _Namen_.